formats/avro/src/test/java/io/cloudevents/avro/AvroFormatTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018-Present The CloudEvents Authors
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.cloudevents.avro;
+
+import java.util.Map;
+import java.net.URI;
+
+import io.cloudevents.CloudEvent;
+import io.cloudevents.CloudEventData;
+import io.cloudevents.SpecVersion;
+import io.cloudevents.core.builder.CloudEventBuilder;
+import io.cloudevents.core.format.EventFormat;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AvroFormatTest {
+
+    Map<String, Object> testData = Map.of("name", "Ning", "age", 22.0);
+
+    @Test
+    public void testSerde() {
```

Comment:
Can we add more test cases?
- Using a CloudEvent v0.3 (`CloudEventBuilder.v03()`)
- passing null as avroCloudEventData to `AvroCloudEventDataWrapper` (we can create a new class `AvroCloudEventDataWrapperTest` for this
- Testing all data types specified in the union type for `data` field (null included): https://github.com/cloudevents/sdk-java/pull/407/files#diff-09cbc79dec7471fa039c796a1fdfc9f13013cb7e9636e23bc32687378f59cb2aR23-R60

Reply:
Ok. I will do it in next few commits.